### PR TITLE
Updated Blueprint > cp access blurb

### PR DESCRIPTION
**Pull request by bc-mkatz**
* Added callout that Blueprint themes are visible in a store’s cp only if acquired before November 2016.
* Rewrote Blueprint text to acknowledge Blueprint's legacy status.
* Added links to Stencil's merchant-oriented and developer docs.
* For Changelog, fixed one 2017 entry and two heading styles.

### CODE REVIEW

**Comment by bc-AlyssNoland** [comment]
Please include screenshots of the changes you've applied with the css

**Comment by bc-mkatz** [comment]
signup-panel

changelog-heads signup-panel

File: source/stylesheets/_custom.scss [hunk 1]
```diff
@@ -127,13 +127,20 @@
     margin-bottom: 0 !important;
   }
 
-  h2, h3, h4, h5, h6 {
+  h2 {
+    /* Isolated 2/17/17, needs extra leading: */
```
Comment (bc-AlyssNoland) [inline source/stylesheets/_custom.scss]: No need to leave this comment in. Include screenshots in the PR.

File: source/stylesheets/_custom.scss [hunk 2]
```diff
@@ -127,13 +127,20 @@
...
+    margin-top: 28px !important;
+    margin-bottom: 22px !important;
+  }
+
+  /* h2,*/  h3, h4, h5, h6 {
```
Comment (bc-AlyssNoland) [inline source/stylesheets/_custom.scss]: No need to leave this comment. Git has historical information.

File: source/stylesheets/_custom.scss [hunk 3]
```diff
@@ -127,13 +127,20 @@
...
     margin-top: 0;
     margin-bottom: 11px;
   }
 
   h3, h4 {
-    padding-top: 2%;
+    /* Removed 2/17/17, exaggerates leading: */
```
Comment (bc-AlyssNoland) [inline source/stylesheets/_custom.scss]: if you are removing any css that affects this, the entire entry should be removed, not left commented out.

File: source/stylesheets/_custom.scss [hunk 4]
```diff
@@ -339,12 +346,20 @@ li a, .tocify-wrapper {
 }
 
 #mc-embedded-subscribe:hover {
-  background-color: #3a426e;
-  color: #fff;
+/*  background-color: #3a426e;
+  color: #fff; */
+/* ^ Conformed to BC colors 2/17/17: */
+  background-color: #4e75f8;
```
Comment (bc-AlyssNoland) [inline source/stylesheets/_custom.scss]: please use variables for the color.

File: source/stylesheets/_custom.scss [hunk 5]
```diff
@@ -339,12 +346,20 @@ li a, .tocify-wrapper {
...
+  color: white;
   border-color: #3a426e;
 }
 
 #mc-embedded-subscribe {
+/* Conformed to BC branding 2/17/17: */
+  background-color: #4e75f8;
```
Comment (bc-AlyssNoland) [inline source/stylesheets/_custom.scss]: please use variables for colors.

**Comment by becomevocal** [comment]
@bc-AlyssNoland Looks like @bc-mkatz has updated this PR based on the feedback. Can you merge?

**Comment by bc-AlyssNoland** [comment]
To clarify, I have yet to merge this because I need to rebase and squash the merge messages in this PR. I haven't had time to do so yet.

**Comment by chrisboulton** [comment]
@bc-mkatz: Can you clean up the commits in this merge request before we merge it? There's a lot of commits above which look empty or unrelated to the changes you're trying to make (both based on the diff and your description).

I tried to attempt to rebase and squash, and things just got messy I was worried your changes would be lost.

**Comment by bc-mkatz** [comment]
@chrisboulton: Please ping me back when you're available, I suggest we resolve the commits together. Thanks.

**Comment by chrisboulton** [comment]
@bc-mkatz So, I sat down and tried to understand what Git wanted to do to try and rebase it, and I couldn't even figure it out.

I'd recommend just taking the changes and reapplying them to a fresh branch based off master, and then we go from there and merge it down. I'd recommend we close this PR. You should be able to do that in one commit, it shouldn't take more than 5 minutes.

In general a few things that might make this easier in the future - instead of working off `master` in a fork, work off a branch that's specific for the changes that are being developed. I think from the commit history, it looks like changes have been merged from the upstream bc-labs repo into your fork on the master branch, but instead I'd use a rebase strategy which takes your changes, undoes them, applies the latest commits, then applies your changes - you end up with a neater looking history and it avoids all the strange merge commits you can see above.

**Comment by bc-mkatz** [comment]
@chrisboulton: I agree with your overall approach, but would appreciate guidance in implementing it. (I need to resolve the fork complications, plus I want to avoid complicating this with some uncommited changes I've since made locally.) May I see you when you're available?

**Comment by bc-AlyssNoland** [comment]
@bc-mkatz @chrisboulton Nate made a separate branch and created a new PR with three commits to resolve this. @becomevocal's PR has been merged and deployed.

**Pull request closed without merging**